backend/README.md

```diff
@@ -1,38 +1,29 @@
-# SSO Pathfinder Confidential Client Using Node-Express
+# URL-S Backend
 
-## Description
+Instructions for configuring and starting the URL-S backend apps: Node.js Express and MongoDB.
 
-This is an example backend app that authenticates users using OAuth 2.0 standard flow
+## Local Setup
+ **`Step 1`**. Make sure you have Node.js installed in your local machine. Type node -v in Terminal. This repo has been tested with Node version 20.10.0 LTS. If you do not have Node, download from here - https://nodejs.org/en/. npm will be installed alongside node.js.
+ 
+ **`Step 2`**. Make sure you have MongoDB installed and running. Install Homebrew and then `brew install mongodb-community@7.0`. Start the database service with `brew services start mongodb-community`
```

Comment:
We need this command before we can install the community edition
`brew tap mongodb/brew`

Also Compass can be installed visa brew 
`brew install --cask mongodb-compass`

Reply:
Fixed with commit = "Update README for mongodb/brew and grew Compass changes"